### ISSUE TITLE
Predict model

### ISSUE THREAD

**Issue by covuworie** [covuworie/nobel-physics-prizes#84]
Train the LR + sample weights model on the full training data (train + validation) and evaluate the performance on the test set. View MCC, confusion matrix and classification report. Compare to the baseline classifier.
